File: Programming-Basics-with-Python/10.While Loop - Exercise/01_old_books.py
wanted_book = input()
current_book = ""
book_counter = 0
while wanted_book != current_book:
    current_book = input()
    if current_book == "No More Books":
        break
    book_counter += 1
if wanted_book == current_book:
    book_counter -= 1
    print(f"You checked {book_counter} books and found it.")
else:
    print("The book you search is not here!")
    print(f"You checked {book_counter} books.")

File: Programming-Basics-with-JavaScript/5.While Loop/01oldBooks.js
function oldBooks(input){
    let i = 0
    let checkedBooks = 0
    let isValid = false
    let books = input[i]
    i++
    let currentBooks = input[i]
    i++
    while (currentBooks !== 'No More Books'){
        checkedBooks += 1
        if (currentBooks == books){
            checkedBooks -= 1
            isValid = true
            break
        }
        
        currentBooks = input[i]
        i++
        
    }
    if (isValid){
        console.log(`You checked ${checkedBooks} books and found it.`)
    }else{
        console.log(`The book you search is not here!\nYou checked ${checkedBooks} books.`
        )

    }
}
oldBooks(["Bourne",
"True Story",
"Forever",
"More Space",
"The Girl",
"Spaceship",
"Strongest",
"Profit",
"Tripple",
"Stella",
"The Matrix",
"Bourne"])



